test_files/decorator/external.js

```diff
@@ -17,3 +17,5 @@ exports.AClassWithGenerics = AClassWithGenerics;
  */
 function AType() { }
 exports.AType = AType;
+function AType_tsickle_Closure_declarations() {
+}
```

Comment:
Re frequency, I don't think it's generally common to define interfaces that contain no values, so this should be rare.